src/Files.App/Views/SettingsPages/Preferences.xaml

```diff
@@ -162,7 +162,14 @@
 								IsOn="{Binding ListAndSortDirectoriesAlongsideFiles, Mode=TwoWay}"
 								Style="{StaticResource RightAlignedToggleSwitchStyle}" />
 						</local:SettingsBlockControl>
-					</StackPanel>
+
+                        <local:SettingsBlockControl Title="{helpers:ResourceString Name=SelectFilesAndFoldersOnHover}" HorizontalAlignment="Stretch">
```

Comment:
I'd move this setting to the "Click to open settings" section below (perhaps changing the section name to "Behavior for opening and selecting files and folders"). @yaichenbaum?

Reply:
That sounds a bit confusing, let's keep it in this section for now.

Reply:
That section would contain how the user interacts with items.. np.

Reply:
I want to eventually remove the files and folders section but I need some time to think it over.